src/main/java/com/thebois/models/world/resources/Stone.java

```diff
@@ -1,34 +1,37 @@
 package com.thebois.models.world.resources;
 
 import com.thebois.models.Position;
+import com.thebois.models.inventory.items.IItem;
+import com.thebois.models.inventory.items.ItemFactory;
+import com.thebois.models.inventory.items.ItemType;
 
 /**
- * Resource of type rock.
+ * A large stone on the ground.
  */
-public class Rock extends AbstractResource {
+public class Stone extends AbstractResource {
 
     /**
      * Instantiate a new rock resource at given position.
      *
-     * @param position The position where the rock resource should be created.
+     * @param x X coordinate for the rock resource.
+     * @param y Y coordinate for the rock resource.
```

Comment:
Oh you're right.

Really, the javadoc shouldn't have to mention the name of the class. It's already apparent that the javadoc is describing the class, because its located inside of the class. Here we see a great example of why to avoid writing out type names in javadoc.

I'll fix it.

Reply:
Sorry

Reply:
It's no problem, it's a very easy mistake to make! It's more natural to write it the way you have, so you kind of have to force yourself to leave out the "for xxxx" at the end.